src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStep.java

```diff
@@ -116,10 +119,13 @@ public boolean canSubmit() {
      */
     @Deprecated
     public boolean canSettle(Authentication a) {
-        if (submitter==null || a.getName().equals(submitter))
+        if (submitter==null)
+            return true;
+        final Set<String> submitters = Sets.newHashSet(submitter.split(","));
+        if (submitters.contains(a.getName()))
```

Comment:
I think being able to add a space after the coma is a usual thing for a user to do to make the code more readable.
Whilst some realms do allow groups with a trailing space - its probably not normal - so could we trim the submitter before checking so we can use
`"bob, jane, freddy`"?

Reply:
Improvement proposed as https://github.com/jenkinsci/pipeline-input-step-plugin/pull/30 @jtnord BTW